### PR TITLE
Add memory test script

### PR DESCRIPTION
**Pull request by davvore33**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,52 +32,8 @@ WARP17 will be able to run on all the platforms that are supported by DPDK.
...
-
-It is interesting to see that when emulating real HTTP traffic on top of
-a few million TCP sessions, WARP17 can easily exhaust the 40Gbps throughput of
-the link.
-
-NOTE: the script used to obtain the benchmark results is available in the
-codebase at `examples/python/test_2_perf_benchmark.py`. The script spawns WARP17
-for each of the test configurations we were interested in.
+You can find all the performance test and the reference architecture
+in the doc folder [here](./doc/Performance.md).
```
Comment (dceara) [inline README.md]: I would put the link on the text, something like:
```
in the [doc folder](./doc/Performance.md).
```

File: README.md [hunk 2]
```diff
@@ -32,52 +32,8 @@ WARP17 will be able to run on all the platforms that are supported by DPDK.
...
-* Ethernet link utilization (percentage of 40Gbps)
-
-It is interesting to see that when emulating real HTTP traffic on top of
-a few million TCP sessions, WARP17 can easily exhaust the 40Gbps throughput of
-the link.
-
-NOTE: the script used to obtain the benchmark results is available in the
-codebase at `examples/python/test_2_perf_benchmark.py`. The script spawns WARP17
-for each of the test configurations we were interested in.
+You can find all the performance test and the reference architecture
```
Comment (dceara) [inline README.md]: Maybe change to:
```
You can find all the performance tests descriptions and the reference hardware architecture details
```

File: README.md [hunk 3]
```diff
@@ -1457,299 +1413,12 @@ sudo perl -I ./perl/ examples/perl/test_1_http_4M.pl
...
-
-* commit messages: please make sure that commit messages follow the
-  `.git-commit.template` provided in the repository. In order to enforce this
-  template locally you can execute the following command:
-
-```
-git config commit.template ./.git-commit.template
-```
+You can find how to contribute to our project and how to add new L7
+application implementations (here)[here](./doc/Contributing.md).
```
Comment (dceara) [inline README.md]: Please remove the duplicate "here".

**Review by dceara — APPROVED** [review]
Please fix the minor issues and merge the PR afterwards.

**Comment by davvore33** [comment]
2018-11-16 14:18:26,281 - TestPerf - INFO - Test test_01_4M_tcp_sess_setup_rate
2018-11-16 14:19:50,174 - TestPerf - INFO - Average Rate 9230273
2018-11-16 14:19:50,174 - TestPerf - INFO - Test test_02_8M_tcp_sess_setup_rate
2018-11-16 14:22:20,680 - TestPerf - INFO - Average Rate 8999138
2018-11-16 14:22:20,681 - TestPerf - INFO - Test test_03_10M_tcp_sess_setup_rate
2018-11-16 14:25:24,912 - TestPerf - INFO - Average Rate 8910223
2018-11-16 14:25:24,912 - TestPerf - INFO - Test test_04_4M_tcp_sess_data_10b_setup_rate
2018-11-16 14:27:00,567 - TestPerf - INFO - Average Rate 3766282
2018-11-16 14:27:00,567 - TestPerf - INFO - Test test_05_4M_tcp_sess_data_1024b_setup_rate
2018-11-16 14:28:35,292 - TestPerf - INFO - Average Rate 2726264
2018-11-16 14:28:35,292 - TestPerf - INFO - Test test_06_4M_tcp_sess_data_1300b_setup_rate
2018-11-16 14:30:10,946 - TestPerf - INFO - Average Rate 3954830
2018-11-16 14:30:10,946 - TestPerf - INFO - Test test_07_4M_udp_sess_data_10b_setup_rate
2018-11-16 14:31:31,812 - TestPerf - INFO - Average Rate 11666678
2018-11-16 14:31:31,813 - TestPerf - INFO - Test test_08_4M_http_sess_data_10b_setup_rate
2018-11-16 14:33:05,955 - TestPerf - INFO - Average Rate 3378078
2018-11-16 14:33:05,956 - TestPerf - INFO - Test test_09_4M_udp_mcast_flows_data_10b_setup_rate
2018-11-16 14:34:28,457 - TestPerf - INFO - Average Rate 22758531
2018-11-16 14:34:28,457 - TestPerf - INFO - Test test_10_timestamp_4M_tcp_sess_setup_rate
2018-11-16 14:36:00,011 - TestPerf - INFO - Average Rate 7978432
2018-11-16 14:36:00,011 - TestPerf - INFO - Test test_11_timestamp_4M_udp_sess_data_10b_setup_rate
2018-11-16 14:37:23,947 - TestPerf - INFO - Average Rate 11044116
2018-11-16 14:37:23,947 - TestPerf - INFO - Test test_12_recent_timestamp_4M_tcp_sess_setup_rate
2018-11-16 14:38:55,026 - TestPerf - INFO - Average Rate 8036528
2018-11-16 14:38:55,027 - TestPerf - INFO - Test test_13_recent_timestamp_4M_udp_sess_data_10b_setup_rate
2018-11-16 14:40:18,744 - TestPerf - INFO - Average Rate 11113622
2018-11-16 14:40:18,744 - TestPerf - INFO - Test test_14_timestamp_raw_4M_tcp_sess_data_1024b_setup_rate
2018-11-16 14:41:53,907 - TestPerf - INFO - Average Rate 2996067
2018-11-16 14:41:53,908 - TestPerf - INFO - Test test_15_timestamp_raw_4M_udp_sess_data_1024b_setup_rate
2018-11-16 14:43:10,410 - TestPerf - INFO - Average Rate 13466451
2018-11-16 14:43:10,410 - TestPerf - INFO - Test test_16_recent_timestamp_raw_4M_tcp_sess_data_1024b_setup_rate
2018-11-16 14:44:45,852 - TestPerf - INFO - Average Rate 3163370
2018-11-16 14:44:45,852 - TestPerf - INFO - Test test_17_recent_timestamp_raw_4M_udp_sess_data_1024b_setup_rate
2018-11-16 14:46:02,336 - TestPerf - INFO - Average Rate 13441268

**Pull request merged**